### PR TITLE
GitHub oauth

### PR DESCRIPTION
**Pull request by bplantico**
What does this PR do?

This pull request tests the OAuth/OmniAuth implementation and also removes the google env variable from the omniauth config since it was not being used.

### CODE REVIEW

File: spec/features/user/user_can_connect_to_github_with_oauth_spec.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+    context 'when I click link to connect to GitHub and complete OAuth process' do
+      it 'my dashboard shows GitHub repos/followers/following' do
+        OmniAuth.config.test_mode = true
+
+        OmniAuth.config.mock_auth[:github] = OmniAuth::AuthHash.new({
+          provider: 'github',
+          uid: '123545',
+          credentials: {token: '765434'}
+        })
```
[the remark this answers is not in the document]
Comment (BrennanAyers) [reply spec/features/user/user_can_connect_to_github_with_oauth_spec.rb:30]: No thank you, Hound.

File: Gemfile.lock [hunk 2]
```diff
@@ -369,4 +373,4 @@ RUBY VERSION
    ruby 2.4.1p111
 
 BUNDLED WITH
-   1.16.2
+   2.0.2
```
Comment (BrennanAyers) [inline Gemfile.lock:376]: Did we mean to do this? @alexander-mathieu @bplantico

File: db/schema.rb [hunk 3]
```diff
@@ -68,6 +68,7 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.string "github_token"
+    t.string "github_uid"
```
Comment (BrennanAyers) [inline db/schema.rb:71]: We'll see if this is necessary later, but good to be future-proofing.

File: spec/features/user/user_can_connect_to_github_with_oauth_spec.rb [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+          expect(@user.github_uid).to eq('123545')
+          expect(@user.github_token).to eq('765434')
+
+          expect(page).to_not have_link('Connect to GitHub')
+
+          within '.github-dashboard' do
+            expect(page).to have_css('.github-dashboard-repos')
+            expect(page).to have_css('.github-dashboard-followers')
+            expect(page).to have_css('.github-dashboard-following')
+          end
```
Comment (BrennanAyers) [inline spec/features/user/user_can_connect_to_github_with_oauth_spec.rb:47]: Good stuff, I think this is covered well enough in our other specs so this is really just the spot check.

File: Gemfile.lock (hunk 2 above)
Comment (alexander-mathieu) [reply Gemfile.lock:376]: Seems fine, if anything goes wrong it'll probably be with Heroku!

**Review by BrennanAyers — APPROVED** [review]
We're on track! Glad we could work together on this Omniauth stuff and get that test sorted.

**Pull request merged**